q2_vsearch/tests/test_cluster_sequences.py

```diff
@@ -145,7 +145,7 @@ class ParseUc(TestPluginBase):
     def test_empty(self):
         """ empty uc file returns empty Table
         """
-        actual = _parse_uc(uc_empty.split('\n'))
+        actual = parse_uc(uc_empty.split('\n'))
```

Comment:
We have this class tagged for removal (in addition to the `_parse_uc` function), as these tests are copied from the biom-format project. I think we should pull the class out now too, since the function is being tested in biom-format.

Reply:
Oh thanks, yeah I should've noticed that before, that's what I get for running a global find-and-replace.

Reply:
@lizgehret - can you finish this PR for me?

Reply:
@thermokarst absolutely! So to clarify, we are just wanting to remove all tests where you had replaced the method _parse_uc() with parse_uc()?

Reply:
Awesome, thanks @lizgehret!

> we are just wanting to remove all tests where you had replaced the method _parse_uc() with parse_uc()?

Yes! I think the entire test class called `ParseUc` can be dropped.

Reply:
@thermokarst copy that! Updated locally on my machine, can you give me permission to push changes up to your fork?